Index: libqaccessibilityclient/CMakeLists.txt
===================================================================
--- libqaccessibilityclient.orig/CMakeLists.txt	2014-02-26 14:56:05.000000000 +0100
+++ libqaccessibilityclient/CMakeLists.txt	2014-02-26 16:35:45.263545936 +0100
@@ -47,7 +47,7 @@
 
 if (WIN32)
    set(EXEC_INSTALL_PREFIX  "")
-   set(LIB_INSTALL_DIR "lib${LIB_SUFFIX}")
+   set(LIB_INSTALL_DIR "lib${LIB_SUFFIX}" CACHE STRING "Directory where lib will install")
    set(BIN_INSTALL_DIR "bin")
    set(INCLUDE_INSTALL_DIR  "include")
 else (WIN32)
@@ -56,7 +56,8 @@
    endif(APPLE)
    set(EXEC_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
    set(BIN_INSTALL_DIR "${EXEC_INSTALL_PREFIX}/bin")
-   set(LIB_INSTALL_DIR "${EXEC_INSTALL_PREFIX}/lib${LIB_SUFFIX}")
+   set(LIB_INSTALL_DIR "${EXEC_INSTALL_PREFIX}/lib${LIB_SUFFIX}" CACHE STRING
+       "Directory where lib will install")
    set(INCLUDE_INSTALL_DIR  "${CMAKE_INSTALL_PREFIX}/include")
 endif (WIN32)
 
